plugins/woocommerce/src/Admin/Features/ProductBlockEditor/Init.php

```diff
@@ -56,6 +57,13 @@ public function __construct() {
 		$this->redirection_controller = new RedirectionController( $this->supported_post_types );
 
 		if ( \Automattic\WooCommerce\Utilities\FeaturesUtil::feature_is_enabled( 'product_block_editor' ) ) {
+			add_filter( 'product_type_selector', function ( $types ) {
+				$types['simple-subscription'] = __( 'Simple subscription', 'woocommerce' );
+				$types['variable-subscription'] = __( 'Variable subscription', 'woocommerce' );
+				return $types;
+			}, 999, 1 );
```

Comment:
Why is this needed?

Subscriptions aren't a part of core.

This affects everywhere `wc_get_product_types()` is used (28 instances from my count, not including extensions).

I'm concerned this would have unexpected consequences.